src/tabris/util-colors.js

```diff
@@ -29,6 +29,24 @@ export function colorStringToArray(str) {
       255
     ];
   }
+  // #xxxxxxxx
+  if (/^#([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})$/.test(str)) {
```

Comment:
Couldn't we save code and cycles by making the last position optional and so combine the regexp for 8 and 6 hex digit colors (and also for 4 and 3 digits)?
```js
/^#([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})([0-9a-fA-F]{2})?$/
```
For 6-digit hex numbers, `RegExp.$4` would then be the empty string.